packit_service/log_versions.py

```diff
@@ -0,0 +1,42 @@
+import logging
+
+from ogr import __version__ as ogr_version
+from packit_service import __version__ as ps_version
+from sqlalchemy import __version__ as sqlal_version
+from flask_restx import __version__ as restx_version
+from flask import __version__ as flask_version
+
+# from packit import __version__ as packit_version
+
+logger = logging.getLogger(__name__)
+
+
+class LogPackageVersions:
```

Comment:
As far as I see, nothing in the code bellow required having a class. Let's keep things simple and use only functions, without having a class.